tests/unit/test_column_group.py

```diff
@@ -57,8 +56,10 @@ def test_nested_column_group():
     assert geo_country_user[0] == geo_country_user[1]  # US / userA
     assert geo_country_user[2] != geo_country_user[0]  # same user but in canada
 
+    # TODO: Sort out how to check this since >>'ing a list in to an op no longer works
+
     # make sure we get an exception if we nest too deeply (can't handle arbitrarily deep
     # nested column groups - and the exceptions we would get in operators like Categorify
     # are super confusing for users)
-    with pytest.raises(ValueError):
-        cats = [[country + "user"] + country + "user"] >> Categorify(encode_type="combo")
```

Comment:
Combo still an issue? Even with grouped names?

Reply:
Yeah, the nested case is hard. As of these commits, the `Categorify(encode_type="combo")` functionality works fine but this particular bit of error checking doesn't, because it's no longer allowed to `>>` a list onto an operator. In subsequent commits that further disentangle `ColumnSelector` from `ColumnGroup` (coming soon), I'm still wrestling with how to represent and error check the nested cases.